src/sql/src/session/vars.rs

```diff
@@ -2415,6 +2423,9 @@ impl SystemVars {
                 .lock()
                 .expect("lock poisoned")
                 .limit = u64::cast_from(*self.expect_value(&MAX_CONNECTIONS));
+        } else if name == VARIABLE_LENGTH_ROW_ENCODING.name {
+            let value = *self.expect_value(&VARIABLE_LENGTH_ROW_ENCODING);
+            mz_repr::VARIABLE_LENGTH_ROW_ENCODING.store(value, atomic::Ordering::SeqCst);
         }
```

Comment:
Is it true that we will apply this change every time LD pushes a variable update? So when the value in LD changes while envd is running that would change the way row encoding is performed in envd? Is that fine?

Reply:
I can't immediately see how it would cause problems, but for sanity, it's probably better if we always keep envd in sync with clusterd. Good catch.